.gitignore

```diff
@@ -12,3 +12,5 @@ gradle-app.setting
 
```

Comment:
It's very good that you've added gitignore file.
Put here everything except for code and .gitignore file itself.